src/controller/request/product-request.ts

```diff
@@ -56,5 +62,8 @@ export interface CreateProductRequest extends BaseProductParams {
  * @property {number} vat.required - VAT group ID of the product
  * @property {number} category.required  - Category of the product
  * @property {number} alcoholPercentage.required  - Alcohol percentage of the product in 2 decimals
+ * @property {boolean} featured.required - If product is featured
```

Comment:
again, I don't think you should have to specify this, but its fine since it will most likely be added to the FE as a checkmark anyhow.

Reply:
I would just put this as false by default. FE implementation on creation doesn't really need to include this right? Just having a toggle on each product entry to feature it would be enough imo.

Reply:
I changed it such that they are always optional to fill in as indeed even at creation or updating you do not always need it.